### PR TITLE
Fix journal dumper not work

### PR DESCRIPTION
**Pull request by qian0817**
### What changes are proposed in this pull request?

fix journal dumper not work.

### Why are the changes needed?

when use command `./bin/alluxio readJournal` will face error like:
```
java.lang.RuntimeException: java.lang.IllegalStateException: Raft journal entries should never set multiple fields in addition to sequence number, but found sequence_number: 6
update_inode {
  id: 0
  last_modification_time_ms: 1641571923648
  last_access_time_ms: 1641571923648
}
operationId {
  mostSignificantBits: 3407999256801922785
  leastSignificantBits: -4799816901488576434
}

        at alluxio.master.journal.tool.RaftJournalDumper.lambda$readRatisLogFromDir$0(RaftJournalDumper.java:99)
        at org.apache.ratis.server.raftlog.segmented.LogSegment.readSegmentFile(LogSegment.java:141)
        at org.apache.ratis.server.raftlog.segmented.LogSegment.readSegmentFile(LogSegment.java:125)
        at alluxio.master.journal.tool.RaftJournalDumper.readRatisLogFromDir(RaftJournalDumper.java:90)
        at alluxio.master.journal.tool.RaftJournalDumper.readFromDir(RaftJournalDumper.java:79)
        at alluxio.master.journal.tool.RaftJournalDumper.dumpJournal(RaftJournalDumper.java:69)
        at alluxio.master.journal.tool.JournalTool.dumpJournal(JournalTool.java:121)
        at alluxio.master.journal.tool.JournalTool.main(JournalTool.java:95)
Caused by: java.lang.IllegalStateException: Raft journal entries should never set multiple fields in addition to sequence number, but found sequence_number: 6
update_inode {
  id: 0
  last_modification_time_ms: 1641571923648
  last_access_time_ms: 1641571923648
}
operationId {
  mostSignificantBits: 3407999256801922785
  leastSignificantBits: -4799816901488576434
}

        at com.google.common.base.Preconditions.checkState(Preconditions.java:589)
        at alluxio.master.journal.tool.RaftJournalDumper.writeSelected(RaftJournalDumper.java:153)
        at alluxio.master.journal.tool.RaftJournalDumper.writeSelected(RaftJournalDumper.java:162)
        at alluxio.master.journal.tool.RaftJournalDumper.lambda$readRatisLogFromDir$0(RaftJournalDumper.java:97)
        ... 7 more
```

### Does this PR introduce any user facing changes?

No user facing changes.

### CODE REVIEW

**Comment by jenoudet** [comment]
How do you reproduce this error?

**Comment by qian0817** [comment]
> How do you reproduce this error?

```
./bin/alluxio formatJournal
./bin/alluxio-start.sh master
./bin/alluxio fs mkdir /a
./bin/alluxio-stop.sh master
./bin/alluxio readJournal
```
After that, an exception will be thrown as above.

File: core/server/master/src/main/java/alluxio/master/journal/tool/RaftJournalDumper.java [hunk 1]
```diff
@@ -151,10 +151,9 @@ private void writeSelected(PrintStream out, Journal.JournalEntry entry) {
       return;
     }
     Preconditions.checkState(
-        entry.getAllFields().size() <= 1
-            || (entry.getAllFields().size() == 2 && entry.hasSequenceNumber()),
+        entry.toBuilder().clearOperationId().clearSequenceNumber().getAllFields().size() <= 1,
```
[the remark this answers is not in the document]
Comment (qian0817) [reply core/server/master/src/main/java/alluxio/master/journal/tool/RaftJournalDumper.java:154]: In #14162, `JournalEntry` add an additional field `OperationId`. Therefore, the OperationId field needs to be clear when determining here.

**Review by jenoudet — APPROVED** [review]
Thanks for the fix!

**Review by yuzhu — APPROVED** [review]

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Pull request merged**